app/controllers/katello/api/v2/packages_controller.rb

```diff
@@ -2,6 +2,9 @@ module Katello
   class Api::V2::PackagesController < Api::V2::ApiController
     apipie_concern_subst(:a_resource => N_("a package"), :resource => "packages")
     include Katello::Concerns::Api::V2::RepositoryContentController
+    include Katello::Concerns::Api::V2::RepositoryDbContentController
+
+    before_filter :find_repositories, :only => :auto_complete_name
 
     api :GET, "/packages", N_("List packages")
     api :GET, "/repositories/:repository_id/packages", N_("List packages")
```

Comment:
I went back and forth about this.  I feel like more than just that would need to be changed, and it might be better to go through the whole app in a separate PR.  Will file an issue if you agree